src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1066,7 +1066,7 @@
     <value>Cannot implicitly convert type '{0}' to '{1}'. An explicit conversion exists (are you missing a cast?)</value>
   </data>
   <data name="ERR_PartialMisplaced" xml:space="preserve">
```

Comment:
>ERR_PartialMisplaced [](start = 14, length = 20)

I think we should rename the error code/identifier as well, to match the new message.

Reply:
Maybe we could just use ERR_BadMemberFlag for all cases and remove ERR_PartialMisplaced?

Reply:
Yes, `ERR_BadMemberFlag` would be fine.

Reply:
ERR_PartialMisplaced has been removed.